apps/deploy-web/src/components/deployments/DeploymentDepositModal.tsx

```diff
@@ -225,10 +242,29 @@ export const DeploymentDepositModal: React.FunctionComponent<DeploymentDepositMo
         }
       ]}
       onClose={onClose}
-      maxWidth="xs"
       enableCloseOnBackdropClick
-      title="Deployment Deposit"
+      title={title}
     >
+      {services.length > 0 && (
+        <div className="mb-3 max-h-[300px] overflow-scroll">
+          {services.map(service => {
+            return (
+              <Alert key={service.title} className="mb-1">
+                <div className="mb-2 text-sm">
+                  <span className="font-bold">{service.title}</span>:{service.image}
+                </div>
+                <div className="flex items-center space-x-4 whitespace-nowrap">
+                  <LeaseSpecDetail type="cpu" className="flex-shrink-0" value={service.profile?.cpu as number} />
```

Comment:
non-blocking: perhaps it's just copy/paste. seems like type casting isn't necessary here
```suggestion
                  <LeaseSpecDetail type="cpu" className="flex-shrink-0" value={service.profile?.cpu} />
```

Reply:
Nice, thanks, fixed. It was indeed copy-paste from `createDeploymentConfirm`.